src/App.tsx

```diff
@@ -1,57 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Autocomplete } from './components/Autocomplete';
+
+const DELAY = 1000;
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+
+  const handlePersonSelect = (person: Person) => {
+    setSelectedPerson(person);
+  };
 
   return (
     <main className="section">
       <h1 className="title">
-        {`${name} (${born} = ${died})`}
+        {selectedPerson ? (
+          `${selectedPerson.name} (${selectedPerson.born} - ${selectedPerson.died})`
```

Comment:
Maybe you need to redeploy the application, because now there `=` instead of `-`.

<img width="549" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/54860641/101cd84d-1083-4ab7-a3a4-9f00893c52d5">